### PR TITLE
Intern Blog Post - ehearne

### PR DESCRIPTION
**Pull request by ehearneRedHat**
### What:

A blog post about my experience as an intern in Kuadrant.

### CODE REVIEW

**Review by david-martin — COMMENTED** [review]
Thanks for writing this @ehearneRedHat 
This is very insightful and easy to read.

Given that this is for the community blog, I'm wondering you could frame it a little differently to put the emphasis on how working on the Kuadrant project shaped your internship journey, and de-emphasise the Red Hat specific side of it a little?
I wouldn't want to diminish that aspect of your internship as it sounds very valuable and worthwhile to you. However, one way to think about this is if someone not in Red Hat was to read the post and wonder what getting involved in the project would be like. 

I don't want to put words in your mouth, so I'd prefer not to put specific suggestions inline.
That being said, in the 'going back in time' section, if this was a time that sparked your interest in open source projects for example, you could mention that as a driver towards working on open source (and hence the desire to do an internship at Red Hat because of what you saw during your time there).

If there was a specific challenge or problem you worked on in Kuadrant that you felt in retrospect helped when you brought it 'full circle' back to the TY program, that would also be good to mention as it would tie it back to Kaudrant again.

The mention of the kuadrant specific work at the end of the post is great by the way.
No harm in including a link to some of these things to show them off.
If you can think of any other kuadrant related activities that stood out while working on the project, please to mention them as well.

**Comment by ehearneRedHat** [comment]
@david-martin feel free to re-review when you have the chance. I have amended the post to focus more on Kuadrant and open source, and added some more items completed during my time here.

File: src/blog/ehearne-intern-journey.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Bringing It Full Circle
+There was also a period in my internship where I got to be at the other side of the transition year programme, where myself and all the other interns spent the week with the students hosting workshops which gave the students software engineering skills and insight into the software industry. It was one of the highlights of my time here, being able to support young students and watch them grow throughout the week, and I am proud of each and every one of them. Working in Kuadrant helped me immensely as the project uses GitHub for project management, allowimg me to show them software development principles the open source way !
```
Comment (david-martin) [inline src/blog/ehearne-intern-journey.md:26]:
```suggestion
There was also a period in my internship where I got to be at the other side of the transition year programme, where myself and all the other interns spent the week with the students hosting workshops which gave the students software engineering skills and insight into the software industry. It was one of the highlights of my time here, being able to support young students and watch them grow throughout the week, and I am proud of each and every one of them. Working in Kuadrant helped me immensely as the project uses GitHub for project management, allowing me to show them software development principles the open source way !
```

**Comment by ehearneRedHat** [comment]
@david-martin hope that typo is now fixed :)

**Review by david-martin — APPROVED** [review]
Excellent. Thanks again @ehearneRedHat 
I'm going to hold off on merging right now for a couple reasons
- the deploy action is currently failing (i have a pr for that)
- i'd like to space this out from the previous blog post that was merged earlier today (I hope that's OK)

**Comment by ehearneRedHat** [comment]
No worries @david-martin . I am happy for you to merge this and take over from here. I hope this story helps to bring more context as to what being an intern means in Kuadrant for the next round. :)

**Comment by david-martin** [comment]
Merging this now ahead of weekend so it's there for the publish date of Sunday 1st September.

**Pull request merged**